src/tests/recipe-descriptions-test.ts

```diff
@@ -97,16 +136,18 @@ store BoxesStore of [Box] 'allboxes' in AllBoxes` : ''}
         options.manifestString || createManifestString(options),
         {loader, memoryProvider, fileName: 'foo.js'});
     const runtime = new Runtime({loader, composerClass: FakeSlotComposer, context, memoryProvider});
-    const arc = runtime.newArc('demo', 'volatile://');
+    const key = Flags.useNewStorageStack ? (id: ArcId) => new VolatileStorageKey(id, '') : 'volatile://';
```

Comment:
you can also use storageKeyPrefixForTest() in handle-for-test.ts for all of these.

Reply:
In general I prefer to avoid non-local test infrastructure, as I think it tends to make the tests more complex and harder to read.

I do buy that this sort of thing is potentially useful while switching from the old stack to the new though.